modules/gltf/gltf_state.h

```diff
@@ -78,6 +78,8 @@ class GLTFState : public Resource {
 	Vector<int> root_nodes;
 	Vector<Ref<GLTFTexture>> textures;
 	Vector<Ref<Texture2D>> images;
+	TypedArray<String> extensions_used;
+	TypedArray<String> extensions_required;
```

Comment:
I prefer builtin types like `Vector<String>` (`PackedStringArray`) where a TypedArray is not needed.

Reply:
Good suggestion, done.

Reply:
Good suggestion, done.